<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhraseAddWindow</class>
 <widget class="QDialog" name="PhraseAddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phrase manager</string>
  </property>
  <widget class="QTextEdit" name="PhraseAddTextSecond">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="PhraseAddTextMain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TextLanguage1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert the phrase you want to learn in the language you are studying here</string>
   </property>
  </widget>
  <widget class="QLabel" name="TextLanguage1_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert the phrase you want to learn in your native language here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButt">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButt">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="WarningLabel">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>245</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>You must fill out both text fields     </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
